src/components/v5/common/ActionSidebar/partials/forms/EditColonyDetailsForm/EditColonyDetailsForm.tsx

```diff
@@ -11,11 +11,13 @@ import { ActionFormBaseProps } from '../../../types';
 import { useDecisionMethods } from '../../../hooks';
 import SocialLinksTable from './partials/SocialLinksTable';
 import DescriptionRow from '../../DescriptionRow';
+import { useShouldShowCreatedIn } from '../hooks';
 
 const displayName = 'v5.common.ActionSidebar.partials.EditColonyDetailsForm';
 
 const EditColonyDetailsForm: FC<ActionFormBaseProps> = ({ getFormOptions }) => {
   const { decisionMethods } = useDecisionMethods();
+  const shouldShowCreatedIn = useShouldShowCreatedIn();
```

Comment:
I'm not sure this hooks brings much value, I think a better way to encapsulate the logic behind showing the `createdIn` field would be to create a reusable field component, similar to `DescriptionRow` for example. If you don't want to do it in this PR, could you just make this check consistent with others, such as in `EditTeamForm`?